cmd/ingress-operator/main.go

```diff
@@ -119,8 +120,10 @@ func main() {
 
 // createDNSManager creates a DNS manager compatible with the given cluster
 // configuration.
-func createDNSProvider(cl client.Client, operatorConfig operatorconfig.Config, dnsConfig *configv1.DNS, platformStatus *configv1.PlatformStatus) (dns.Provider, error) {
+func createDNSProvider(cl client.Client, operatorConfig operatorconfig.Config, dnsConfig *configv1.DNS, platformStatus *configv1.PlatformStatus, version string) (dns.Provider, error) {
 	var dnsProvider dns.Provider
+	userAgent := fmt.Sprintf("OpenShift/%s (ingress-operator)", version)
```

Comment:
Can we use this for all handlers for better consistency and code re-use?

Reply:
Probably, in a followup though